app/sass/_casestudy.scss

```diff
@@ -0,0 +1,111 @@
+$highlight:#5FC3AD;
```

Comment:
It’s a small thing, but let’s lowercase the hex-code values. See 10 bullets down [from this list](http://codeguide.co/#css-syntax).